pom.xml

```diff
@@ -90,7 +90,7 @@
 
     <modules>
         <module>starter</module>
-        <module>extra-routes</module>
```

Comment:
no greetservice ?

Reply:
Pushed a version where it is there as well

Reply:
Forgot to add it as compiled that module manually during development...